packages/stems/src/components/Scrubber/Slider.tsx

```diff
@@ -219,6 +221,15 @@ export const Slider = ({
     }
   }, [isPlaying, dragPercent, play, pause])
 
+  const playbackRateRef = useRef<number | null>(null)
+
+  useEffect(() => {
+    if (playbackRate !== playbackRateRef.current) {
```

Comment:
Another option is to update the dependencies of this effect to only change when playback rate changes, simulating useEventEffect that's coming out soon?

Reply:
yea I run into this type of thing a lot, lots of ignore-exhaustive-deps. I'll go with this option though.